overlord/configstate/configcore/picfg.go

```diff
@@ -111,21 +133,21 @@ func piConfigFile(dev sysconfig.Device, opts *fsOnlyContext) (string, error) {
 		} else {
 			// we don't support configuring pi-config in these modes as it is
 			// unclear what the right behavior is
-			return "", errPiConfigNotSupported
+			return "", newPiConfigNotSupportedError("unsupported mode")
 		}
 	}
 	return filepath.Join(rootDir, subdir, "config.txt"), nil
 }
 
 func handlePiConfiguration(dev sysconfig.Device, tr config.ConfGetter, opts *fsOnlyContext) error {
 	configFile, err := piConfigFile(dev, opts)
-	if err != nil && err != errPiConfigNotSupported {
-		return err
-	}
-	if err == errPiConfigNotSupported {
-		logger.Debugf("ignoring pi-config settings mode where pi-config changes are unsupported")
+	if _, ok := err.(*piConfigNotSupportedError); ok {
+		logger.Debugf("ignoring pi-config settings: %v", err)
```

Comment:
Maybe this should be a Noticef right now?

Reply:
Indeed, I think that is now a better choice